packages/orchestration-v2/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@sap-ai-sdk/orchestration-v2",
+  "version": "1.17.0",
+  "description": "",
+  "license": "Apache-2.0",
+  "keywords": [
+    "sap-ai-sdk",
+    "orchestration",
+    "llm-access"
+  ],
+  "type": "module",
```

Comment:
[req] Set `private:true` here so we don't accidentally release it to npm with `next`